src/businessservice/src/main/java/com/eventty/businessservice/domains/event/application/serviceImpl/EventServiceImpl.java

```diff
@@ -1,20 +1,39 @@
-package com.eventty.businessservice.domains.event.application;
+package com.eventty.businessservice.domains.event.application.serviceImpl;
 
+import com.eventty.businessservice.domains.event.application.dto.EventResponseDTO;
+import com.eventty.businessservice.domains.event.application.service.EventService;
+import com.eventty.businessservice.domains.event.domain.EventEntity;
 import com.eventty.businessservice.domains.event.domain.EventRepository;
+import com.eventty.businessservice.domains.event.exception.EventNotFoundException;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
+import java.util.List;
+import java.util.stream.Collectors;
+
 @Service
 @RequiredArgsConstructor
 @Transactional
 public class EventServiceImpl implements EventService {
 
     private final EventRepository eventRepository;
 
-
     @Override
-    EventResponseDTO findEventById(Long id){
+    public EventResponseDTO findEventById(Long id){
```

Comment:
MyBatis 을 사용해서 Optional 은 안사용하고 있었는데, 3.5 버전 부터는 지원하고 있다고 하네요!
Mybatis supports Optional return type officially now since 3.5.0, refer to http://blog.mybatis.org/2019/01/mybatis-350-released.html

- 그래서 바꿔서 테스트해봤는데, 테스트코드 (@MybatisTest, MyBatis 에서 지원하는 Repository 단위테스트) 쪽에서 MyBatis 관련 문제들이 좀 생기는 것 같습니다..ㅠ Optional 안써서 가독성은 떨어진다는 아쉬운 점은 있지만, MyBatis 지원이 아직 불안정한 것 같아서 저희 이 쪽 서버에서는 Optional 사용하지 않고 가는게 어떨까요?